packages/e2e-tests/persistent-seeder-deployment/persistent-seeder.dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM node:10.16.3
```

Comment:
On circle, we use the `circleci/node:10.16.3-browsers` image and then skip the chrome download commands below. Could we pull that trick here, too? Might speed things up. 

Also, can we now delete the existing Dockerfile in this package?